scripts/freefood.coffee

```diff
@@ -0,0 +1,52 @@
+# Description:
+#   Announce free food.
+#
+# Commands:
+#   <botname> free <food> <location>
+
+module.exports = (robot) ->
+  robot.respond /free (.+)/i, (msg) ->
```

Comment:
What you have works quite well, you'll still need the magic removal of in/the/at stuff, but match groups are just a good thing to know :+1: